server/routes/induction/common/highestLevelOfEducationController.ts

```diff
@@ -37,6 +37,30 @@ export default abstract class HighestLevelOfEducationController extends Inductio
     )
     return res.render('pages/induction/prePrisonEducation/highestLevelOfEducation', { ...view.renderArgs })
   }
+
+  highestLevelOfEducationDoesNotRequireQualifications = (
```

Comment:
I've moved these 2 methods into the superclass here (from the Update class), so that both the Create and Update class can use them